src/com/mostc/pftt/util/HostEnvUtil.java

```diff
@@ -490,6 +497,17 @@ private static boolean installedVCRT16(FileSystemScenario fs, String dllFile)
 	protected static void doInstallVCRT(ConsoleManager cm, FileSystemScenario fs, AHost host, String name, String installerFile) throws IllegalStateException, IOException, Exception {
 		String local_file = LocalHost.getLocalPfttDir() + installerFile;
 		String remote_file = local_file;
+		String rule = name.replace(' ', '_');
+
+		ExecOutput op = host.execOut("netsh advfirewall firewall show rule name=" + rule, AHost.ONE_MINUTE);
```

Comment:
This check should be in `addRuletoFirewall`, thus eliminating the duplicated code. Thanks.

Reply:
Updated to eliminate redundant code.